### PR TITLE
Update urls github.com/uber -> github.com/visgl

### PR DESCRIPTION
**Pull request by 1chandu**
For #

#### Background

Update links to point to new visgl repo.

/github.com/uber/deck.gl -> /github.com/visgl/deck.gl
/github.com/uber/luma.gl -> /github.com/visgl/luma.gl
/github.com/uber/react-map-gl ->  /github.com/visgl/react-map-gl
/github.com/uber-web/loaders.gl ->  /github.com/visgl/loaders.gl
/uber-common/deck.gl-data -> /visgl/deck.gl-data

#### Change List
-

### CODE REVIEW

**Comment by tsherif** [comment]
@1chandu Did you check if there are links to `uber-common/deck.gl-data`?

File: dev-docs/RFCs/proposals/pydeck-module-loading-rfc.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ commonjs and AMD are two ways to package up JS modules so that they can be impor
 
 ### Dynamic script loading
 
-loaders.gl has implemented a [dynamic script loading system](https://github.com/uber-web/loaders.gl/tree/master/modules/loader-utils/src/lib/library-utils) to support optional, on-demand loading of large loader libraries and worker threads.
+loaders.gl has implemented a [dynamic script loading system](https://https://github.com/visgl/loaders.gl/tree/master/modules/loader-utils/src/lib/library-utils) to support optional, on-demand loading of large loader libraries and worker threads.
```
Comment (kylebarron) [inline dev-docs/RFCs/proposals/pydeck-module-loading-rfc.md:32]: You have an extra `https://` here, and in at least some other places

Comment (kylebarron) [reply dev-docs/RFCs/proposals/pydeck-module-loading-rfc.md:32]: It looks like the changes from the most recent commit have an extra `https://`

**Comment by 1chandu** [comment]
> @1chandu Did you check if there are links to `uber-common/deck.gl-data`?

@tsherif , no I haven't I will update `uber-common/deck.gl-data` to`visgl/deck.gl-data` thansk !!

File: dev-docs/RFCs/proposals/pydeck-module-loading-rfc.md (hunk 1 above)
Comment (1chandu) [reply dev-docs/RFCs/proposals/pydeck-module-loading-rfc.md:32]: Thanks will fix this.

**Pull request merged**